### DISCUSSION TITLE
NOUS A5T - active power stuck in 1

### DISCUSSION THREAD

**Discussion by gilcu3** [arendst/Tasmota#21803] [Q&A] [answered]
I recently bought a Nous power strip from amazon, A5T model, which comes with tasmota preinstalled.

Tasmota version: `13.2.0`

![image](https://github.com/user-attachments/assets/545d4a13-350d-463a-b05a-1f89566c98f4)
The problem is that Active Power is always constant at 1W. I have currently connected there a pi5, a fritz router and a tapo camera, therefore the power load should be much higher. I am yet to do the power calibration, but that the value does not change does not sound right to me.

PS: I just increased the load in the pi5, and the Active Power changed to 2. Does that mean that it cannot measure floating point values? This seems pretty weird to me.

**Comment by arendst** [comment, marked as answer]
Start by doing some homework:

- https://tasmota.github.io/docs/Power-Monitoring-Calibration/
- https://tasmota.github.io/docs/Commands/#power-monitoring

Succes.

**Reply by gilcu3** [reply]
Indeed, after posting the question I did the homework and it kind of worked.

For future people reading this, it seems on this device the default `WattRes` is 0, which gives this integer valued `Active Power`
